core/templates/services/i18n-language-code.service.ts

```diff
@@ -36,6 +36,8 @@ export class I18nLanguageCodeService {
   static languageCodeChangeEventEmitter = new EventEmitter<string> ();
   static languageCode: string = AppConstants.DEFAULT_LANGUAGE_CODE;
   static rtlLanguageCodes: readonly string[] = AppConstants.RTL_LANGUAGE_CODES;
+  static hackyTranslationKeys: readonly string[] =
```

Comment:
Add a comment explaining what this means. Assume the person reading your comment hasn't and won't read your design doc.

Reply:
As this a constant, please CAPITALIZE the name.

Reply:
As this is a constant, please CAPITALIZE the name. Why does this property need to be static?

Reply:
Thanks! @ashutoshc8101 capitalized the name, as for static as the above ones were static due to them being used in angularJS and angular I had also made it static but now have made it private.